proposals/inline-classes.md

```diff
@@ -128,31 +213,79 @@ Since boxing doesn't have side effects as is, it's possible to reuse various opt
 ### Type mapping on JVM
 
 Depending on the underlying type of inline class and its declaration site, inline class type can be mapped to the underlying type or to the 
-wrapper type.  
-
+boxed type.  
```

Comment:
Annotate each parameter using an explicit annotation with a boxed type can really improve code readability.
This code should be injected by compiler in the same manner of `@Nullable` annotation.